### ISSUE TITLE
Nixie tubes appear white in 0.15.22

### ISSUE THREAD

**Issue by pfrenssen** [justarandomgeek/nixie-tubes#14]
Since yesterday's update all nixie tubes that do not use the "Use colors" option appear white instead of orange. When I downgrade to 0.15.21 they get their nice glowy orange color again.

From looking at the code this seems to happen because at the very start of a game the `onTickController()` immediately sends a `"keepcolor"` command which causes the default value of `{r=1.0,  g=1.0,  b=1.0, a=1.0}` to persist on the nixie.

Also here the second line will never be executed, since `color` has been instantiated and assigned a value:
```
        local color = newcolor=="keepcolor" and obj.color or newcolor
        if not color then color = {r=1.0,  g=0.6,  b=0.2, a=1.0} end
```

I tried to debug it myself but I didn't manage to output the value of `color` correctly to the console, if I try something like the following then the value is not printed.
```
game.print(serpent.block(color))
```

 I couldn't find a good resource on the Factorio website on how to properly debug the contents of variables :(

**Comment by justarandomgeek** [comment]
I'm aware of this issue (it's a problem with the deflicker that i just added to the color code), i just haven't had a chance to fix it yet. Expect a fix this weekend!

**Comment by justarandomgeek** [comment]
Fixed in 0.15.23

**Comment by pfrenssen** [comment]
Thanks!
